### PR TITLE
first draft of diagrams

### PR DESCRIPTION
**Pull request by mrbojangles3**

### CODE REVIEW

File: docs/install-upgrade/build-wiring.md [hunk 1]
```diff
@@ -107,14 +177,105 @@ Fabric connections serve as connections between switches, they form the fabric o
...
+    S2
+    end
+
+    subgraph VPC 2
+    S3
+    S4
+    end
+
+
+
```
Comment (qmonnet) [inline docs/install-upgrade/build-wiring.md:236]:
```suggestion
```

File: docs/install-upgrade/build-wiring.md [hunk 2]
```diff
@@ -107,14 +177,105 @@ Fabric connections serve as connections between switches, they form the fabric o
...
+    L1([Leaf 1])
+    S1[Server1]
+    S2[Server2]
+    S3[Server3]
+    S4[Server4]
+
+    L1 <-.2,loopback.-> L1;
+    L1 <-.3.-> S1;
+    L1 <--> S2 & S4;
+    L1 <-.1.-> S3;
```
Comment (qmonnet) [inline docs/install-upgrade/build-wiring.md:223]: Could this diagram have explanations for arrows 1, 2, 3, like you added below the last diagram on the page?

**Review by qmonnet — COMMENTED** [review]
Nice!

File: docs/install-upgrade/build-wiring.md (hunk 2 above)
Comment (mrbojangles3) [reply docs/install-upgrade/build-wiring.md:223]: done!

**Review by Frostman — APPROVED** [review]

**Review by qmonnet — APPROVED** [review]

**Pull request merged**